lecture-variables.html

```diff
@@ -118,12 +132,54 @@
 someNumber = someNumber / x
 someNumber /= x
 </code></pre>
+    <p>
+(Optional) <b>Challenge</b>: what is the value of <code>someNumber</code> at the end of the above code snippet? Walk through each line of code and try to compute the value of <code>someNumber</code> yourself. Then, paste the snippet above into codepen to check your answer.
+    </p>
+
+    <h2>Variable Types</h2>
```

Comment:
I'm down with this. Do you think it makes sense to have this at the end? I do have a (admittedly small) bullet about types up on line 81 above...